UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.LoadTests/start.js

```diff
@@ -0,0 +1,91 @@
+import http from "k6/http";
+import { htmlReport } from "https://raw.githubusercontent.com/benc-uk/k6-reporter/main/dist/bundle.js";
+import { textSummary } from "https://jslib.k6.io/k6-summary/0.0.1/index.js";
+import { authenticateUsingAzure } from './oauth/azure.js';
+
+const runTestProductIdentifier = require('./LoadTestForProductIdentifier.js');
+const config = JSON.parse(open('./config.json'));
+const dataHelper = require('./dataHelper.js');
+const productIdentifierData_Small = dataHelper.GetProductIdentifierDataforSmallExchangeSet();
+const productIdentifierData_Medium = dataHelper.GetProductIdentifierDataforMediumExchangeSet();
+const productIdentifierData_Large = dataHelper.GetProductIdentifierDataforLargeExchangeSet();
+
+let clientAuthResp = {};
+
+export let options = {
+    scenarios: {
+        ESSCreationSmallExchangeSet: {
+            exec: 'ESSCreationSmallExchangeSet',
+            executor: 'per-vu-iterations',
+            startTime: '10s',
+            gracefulStop: '5s',
+            vus: 25,
+            iterations: 161,
+            maxDuration: '1h'
+        },
+        ESSCreationMediumExchangeSet: {
+            exec: 'ESSCreationMediumExchangeSet',
+            executor: 'per-vu-iterations',
+            startTime: '10s',
+            gracefulStop: '5s',
+            vus: 5,
+            iterations: 161,
+            maxDuration: '1h'
+        },
+        ESSCreationLargeExchangeSet: {
+            exec: 'ESSCreationLargeExchangeSet',
+            executor: 'per-vu-iterations',
+            startTime: '10s',
+            gracefulStop: '5s',
+            vus: 1,
+            iterations: 170,
```

Comment:
This may be valid but why is iterations 170 for large but 161 for small and medium?

Reply:
Hi Greg, This is for sampling the overall requests to make total count 5000 requests/hour.